includes/api/class-wc-rest-products-controller.php

```diff
@@ -1702,8 +1704,7 @@ protected function update_post_meta_fields( $post, $request ) {
 					$this->save_variations_data( $product, $request );
 				} else {
 					// Just sync variations.
-					WC_Product_Variable::sync( $product->get_id() );
-					WC_Product_Variable::sync_stock_status( $product->get_id() );
+					WC_Product_Variable::sync( $product, true );
```

Comment:
@mikejolley That wasn't me - it was from your PR the other day #12277. 

https://github.com/woocommerce/woocommerce/pull/12277/files#diff-9d28960c60468fc038da5e6232cf0d8cR709